lib/rules/require-capitalized-constructors.js

```diff
@@ -11,6 +11,9 @@
  *
  * ```js
  * "requireCapitalizedConstructors": true
+ * "requireCapitalizedConstructors": [
```

Comment:
Brainstorming is definitely appreciated and welcomed. However, the new config format has been agreed upon by the community and maintainers, so let's stick with using the `allExcept` format unless @markelog feels there's another format from #698 that makes more sense.

Thanks for the PR @samlecuyer. I'll do a more careful review once the config change is done. Feel free to ping me.